xml/art_sle_ha_pmremote.xml

```diff
@@ -327,28 +517,24 @@
         The key for the &pmremote; service is different from the cluster
         authentication key that you create in the &yast; cluster module.</para>
       </step>
-      <!--
       <step>
-       <para>Restart Pacemaker:</para>
-       <remark>toms 2017-04-13: Is this really needed?</remark>
-       <screen>&prompt.root;<command>systemctl</command> restart pacemaker</screen>
+       <para>In <filename>/etc/csync2/csync2.cfg</filename>, add the
+        following line:</para>
+       <screen>include /etc/pacemaker/authkey</screen>
       </step>
-      -->
       <step>
        <para>In <filename>/etc/csync2/csync2.cfg</filename>, add the
         following line:</para>
        <screen>include /etc/pacemaker/authkey</screen>
```

Comment:
This step is redundant. It's supposed to be "List all cluster nodes and guest nodes in the Csync2 configuration file." which is similar to 3.1.3 I guess ;-)